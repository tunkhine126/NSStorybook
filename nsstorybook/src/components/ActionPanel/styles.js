import { makeStyles } from '@material-ui/core/styles';
import colors from 'global-styles';

export const actionPanelStyles = makeStyles(theme => ({
  cancelBtn: {
    color: colors.TEXT.cancel,
    marginTop: 10,
    '&:hover': {
      color: colors.STATUS.negative,
      backgroundColor: 'transparent',
    },
  },
  panel: {
    background: '#D7D7D7',
    padding: '25px 40px',
    height: '100%',
    [theme.breakpoints.down('md')]: {
      padding: '25px 35px',
    },
  },
  confirmBtn: {
    minHeight: 48,
    width: '100%',
    maxWidth: 225,
    boxShadow: 'none',
    borderRadius: 2,
    fontFamily: 'Barlow',
  },
  progress: {
    color: colors.INTERFACE.ui1,
    padding: 8,
    marginTop: -3,
  },
}));
