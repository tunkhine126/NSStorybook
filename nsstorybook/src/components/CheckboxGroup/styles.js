import { makeStyles } from '@material-ui/core/styles';
import styled from 'styled-components';
import colors from 'global-styles';

export const styles = makeStyles(() => ({
  caption: {
    color: colors.INTERFACE.ui5,
    fontSize: 11,
    fontWeight: 500,
  },
  listItemChkbox: {
    marginLeft: -9,
  },
  listItemText: {
    margin: 0,
  },
  resultsCaption: {
    color: colors.TEXT.light,
  },
}));

export const listItemTextStyles = (multi = false) =>
  makeStyles(() => ({
    primary: {
      color: colors.INTERFACE.ui7,
      fontSize: 12,
      fontWeight: 'normal',
    },
    root: {
      marginTop: 2,
      maxWidth: !multi ? 'auto' : 149,
    },
  }));

export const CheckBoxWrapper = styled.span`
  .MuiCheckbox-root {
    height: 22px;
  }
  .MuiCheckbox-root .MuiIconButton-label:hover {
    width: auto;
  }
`;
