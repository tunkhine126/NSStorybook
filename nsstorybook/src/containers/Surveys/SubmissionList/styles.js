import { makeStyles } from '@material-ui/core/styles';
import colors from 'global-styles';
import styled from 'styled-components';
import { green } from '@material-ui/core/colors';

export const submissionListStyles = makeStyles(() => ({
  actionBtns: {
    paddingRight: '16px',
    paddingBottom: '8px',
  },
  activityContainer: {
    marginRight: '40px',
    flexBasis: '21%',
  },
  avatar: {
    '&.MuiChip-avatar': {
      width: 17,
      height: 17,
    },
  },
  addBtn: {
    padding: 0,
    margin: '15px 0 15px 15px',
    fontSize: 12,
  },
  appBar: {
    backgroundColor: 'transparent',
    borderBottom: `1px solid ${colors.INTERFACE.ui4}`,
    boxShadow: 'none',
  },
  chip: {
    height: 26,
    backgroundColor: 'rgba(0, 0, 0, 0.07)',
    color: colors.TEXT.normal,
    fontWeight: 400,
  },
  container: {
    marginTop: '30px',
  },
  colContainer: {
    overflowWrap: 'break-word',
  },
  columnPaper: {
    outline: 'none',
    borderRadius: 2,
    width: 300,
  },
  columnActionBtns: {
    '& svg': {
      fontSize: 16,
    },
    '& .edit-recipient button, .access-time button': {
      display: 'none',
      padding: 4,
    },
  },
  editBtn: {
    marginBottom: 10,
  },
  columnHeight: {
    height: '35px',
  },
  caption: {
    fontSize: '12px',
    letterSpacing: '0.5px',
    color: colors.TEXT.light,
    fontWeight: 500,
  },
  cardContainer: {
    maxWidth: '215px',
  },
  contentBody: {
    color: colors.INTERFACE.ui7,
  },
  captionSpacing: {
    marginBottom: '5px',
  },
  column1: {
    flexBasis: '31.05%',
  },
  column2: {
    flexBasis: '39.5%',
  },
  columnAction: {
    padding: 0,
    fontSize: '12px',
    textAlign: 'inherit',
  },
  divider: {
    marginLeft: '15px',
  },
  exportBtn: {
    width: '.7em',
    marginRight: '5px',
  },
  expand: {
    color: '#fff',
  },
  modal: {
    display: 'flex',
    alignItems: 'baseline',
    justifyContent: 'center',
    marginTop: 200,
  },
  tab: {
    color: `${colors.TEXT.medium}`,
    letterSpacing: '1px',
    marginRight: '20px',
  },
  recipientLink: {
    padding: 0,
    fontWeight: 600,
  },
  fieldHistory: {
    maxHeight: 437,
    overflow: 'auto',
  },
  familyText: {
    textTransform: 'none',
    letterSpacing: '0',
    padding: 0,
    margin: 0,
    fontSize: '14px',
  },
  recipientName: {
    fontSize: '28px',
    letterSpacing: '1px',
    fontWeight: '600',
    fontFamily: 'Barlow, Roboto, Arial',
    color: colors.TEXT.normal,
    marginBottom: '8px',
  },
  recipientId: {
    fontSize: '10px',
    letterSpacing: '.75px',
    textTransform: 'uppercase',
    color: colors.TEXT.light,
  },
  submissionListHeader: {
    marginBottom: '23px',
  },
  recipientInfo: {
    marginTop: '15px',
  },
  fullWidth: {
    width: '100%',
  },
  familyCommunity: {
    marginTop: '-3px',
  },
  headerImg: {
    height: '153px',
    marginRight: '20px',
    borderRadius: 0,
    boxShadow: 'none',
  },
  recipientChildImg: {
    height: 91,
    '&.MuiCardActionArea-root': {
      width: 135,
    },
    '&:hover .img-action-text': {
      visibility: 'visible',
      opacity: 1,
    },
    '&.MuiCardActionArea-root:hover .MuiCardActionArea-focusHighlight': {
      opacity: 0.5,
    },
  },
  imgActionText: {
    position: 'absolute',
    top: 10,
    bottom: 0,
    left: 0,
    right: 0,
    color: '#fff',
    fontSize: 15,
    fontWeight: 600,
    textAlign: 'center',
    visibility: 'hidden',
    opacity: 0,
    zIndex: 1,
  },
  navContainer: {
    flex: 1,
    marginRight: '40px',
  },
  moreBtn: {
    marginLeft: '15px',
  },
  moveComplete: {
    backgroundColor: '#EAFFEC',
  },
  moveInProgress: {
    backgroundColor: '#fff7dd',
  },
  moveInContainer: {
    padding: '15px',
  },
  movedInDetails: {
    color: colors.TEXT.normal,
    marginTop: '15px',
    fontSize: '13px',
  },
  newFamilyBtn: {
    marginRight: '3px',
  },
  statusProgress: {
    backgroundColor: colors.INTERFACE.ui1,
    borderRadius: 2,
  },
  paper: {
    borderRadius: 2,
    boxShadow: 'none',
    marginBottom: 20,
  },
  photoContainer: {
    fontSize: 12,
    color: colors.TEXT.light,
  },
  checkbox: {
    color: '#fff',
    margin: 'auto',
  },
  recipientChildPhoto: {
    marginLeft: 15,
    marginBottom: 15,
    maxWidth: 200,
  },
  recipientDeleteBtnText: {
    fontSize: 12,
    letterSpacing: 1,
    fontWeight: 700,
    color: colors.STATUS.negative,
  },
  progressTip: {
    marginTop: '-1px',
  },
  twoColumns: {
    padding: '15px 0 15px 15px',
  },
  threeRowTitle: {
    fontSize: '14px',
    fontWeight: '600',
    color: colors.TEXT.normal,
  },
  success: {
    backgroundColor: green[600],
  },
  survyReqBtn: {
    padding: '6px 18px 6px 5px',
  },
  selectedItemsContainer: {
    display: 'flex',
    alignItems: 'center',
    justifyContent: 'space-between',
    height: '40px',
    width: '100%',
    backgroundColor: colors.LINKS.link,
    boxShadow: '0px 2px 4px -3px rgba(88, 88, 88, 1)',
  },
  widgetHeader: {
    fontSize: '16px',
    fontWeight: 'bold',
    letterSpacing: '0.5px',
    marginBottom: '15px',
    color: colors.TEXT.normal,
  },
  widgetPadding: {
    padding: '15px 15px 0 15px',
  },
  upArrow: {
    transform: 'rotate(180deg)',
    width: 10,
  },
  upArrowBtn: {
    padding: 7,
    marginRight: 5,
    '&:hover.MuiIconButton-colorPrimary:hover svg path': {
      fill: colors.LINKS.link,
    },
  },
}));

export const helperClasses = {
  survyReqBtn: {
    padding: '6px 18px 6px 5px',
  },
  lookUpBtn: {
    padding: '10px 0',
  },
  exportBtn: {
    padding: '6px 11px',
  },
  newFamilyBtn: {
    padding: '6px 15px 6px 5px',
  },
  selectorDimension: {
    width: 'auto',
    outterWidth: '107px',
    outterHeight: '0',
  },
  surveysCompleted: {
    outterWidth: '122px',
  },
  actionBtns: {
    padding: '7px 13px',
  },
  createBtn: {
    bg: colors.LINKS.link,
  },
  cancelBtn: {
    bg: 'transparent',
  },
};

export const SelectWrapper = styled.div`
  margin-bottom: 8px;
  .MuiFormLabel-root {
    font-weight: normal;
  }
`;

export const SearchIconStyle = styled.span`
  svg {
    fill: ${colors.TEXT.light};
  }
`;

export const UpArrowStyle = styled.span`
  svg path {
    fill: ${colors.TEXT.light};
  }
`;

export const SelectFormatStyle = styled.div`
  .MuiInputLabel-outlined {
    transform: translate(10px, 13px) scale(1);
  }
  .MuiInputLabel-outlined.MuiInputLabel-shrink {
    transform: translate(12px, 8px) scale(0.75);
  }
  .MuiInputLabel-outlined {
    color: ${colors.TEXT.light};
    font-weight: 400;
  }
  label {
    font-size: 12px;
    letter-spacing: 0;
  }
  input {
    font-size: 13px;
  }
`;

export const SubmissionListStyle = styled.div`
  h2 {
    font-weight: 600;
    font-family: Barlow, Roboto, Helvetica, Arial, sans-serif;
  }
  margin-right: 40px;
  margin-top: 12px;
`;

export const ExpansionPanelDetailsStyle = styled.div`
  .MuiExpansionPanelDetails-root {
    padding: 0;
  }

  .MuiSvgIcon-root {
    transition: none;
  }

  .recipient-delete-btn:hover svg.MuiSvgIcon-root {
    fill: #7d340c;
    transition: fill 0.2s;
  }

  .recipient-delete-btn:hover p {
    color: #7d340c;
    transition: color 0.2s;
  }
  .recipient-delete-btn.MuiButton-contained:active {
    box-shadow: none;
  }
`;

export const ComponentStyle = styled.div`
  .MuiTab-root {
    min-width: 100px;
  }
`;

export const ThreeRowsStyle = styled.span`
  .MuiButton-root {
    text-transform: none;
    font-size: 14px;
  }
`;

export const TabContentStyle = styled.div`
  .field-container.photo,
  .field-container .recipient-column {
    padding: 10px 10px 10px 15px;
    overflow: inherit;
    word-break: break-all;
    min-height: 78px;
  }

  .field-container .recipient-column:hover,
  .field-container.photo:hover {
    transition: background-color 0.3s;
    background-color: ${colors.STATE.subtle};
    justify-content: center;
  }

  .field-container .photo-edit-mode {
    background-color: ${colors.STATE.subtle};
  }

  .field-container .recipient-column:hover .MuiGrid-item {
    width: fit-content;
  }

  .field-container .recipient-column:hover .column-action-btns button {
    display: flex;
  }
  .field-container .edit-cancel-btn {
    font-size: 12px;
    color: ${colors.TEXT.cancel};
  }
  .field-container .edit-cancel-btn:hover {
    color: ${colors.STATUS.negative};
  }
  .field-container .edit-save-btn {
    padding: 6px 12px;
  }
`;

export const ExpansionPanelStyle = styled.div`
  .MuiExpansionPanelSummary-expandIcon.Mui-expanded {
    transform: rotate(90deg);
  }
  .MuiExpansionPanelSummary-root {
    padding: 0 12px 0 15px;
    min-height: 73px;
  }
  .MuiExpansionPanel-root.Mui-expanded,
  .MuiExpansionPanelSummary-content.Mui-expanded {
    margin: 0;
  }
  .MuiPaper-elevation1 {
    box-shadow: none;
  }
  .MuiExpansionPanel-root:before {
    opacity: 0;
  }
  .MuiFormControl-marginNormal {
    margin-top: 0;
    height: 60px;
  }
  .paragraph-text .MuiFormControl-marginNormal {
    height: auto;
  }
  .MuiFormHelperText-root.MuiFormHelperText-contained.Mui-error {
    display: none;
  }
`;
