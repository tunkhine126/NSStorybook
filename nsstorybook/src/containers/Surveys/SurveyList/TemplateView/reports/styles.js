import { makeStyles } from '@material-ui/core/styles';
import colors from 'global-styles';

export const styles = makeStyles(() => ({
  divider: { marginBottom: 20, marginTop: 20 },
  chartContainer: { paddingRight: 50, paddingLeft: 40 },
  coordinates: { maxHeight: 40 },
  locationText: { fontWeight: 600, marginTop: 5, marginBottom: 5 },
  pin: {
    cursor: 'pointer',
    fill: '#d00',
    stroke: 'none',
    marginRight: 5,
  },
  pinLinks: {
    color: colors.TEXT.dark,
    textDecoration: 'underline',
    fontSize: 12,
    marginLeft: 20,
    marginBottom: 5,
    '&:hover': {
      color: colors.STATE.active,
    },
  },
  pinIcon: { fontSize: 13, marginBottom: -3 },
  numberType: {
    color: colors.INTERFACE.ui7,
    marginLeft: 40,
    marginBottom: 5,
    fontWeight: 400,
  },
  numberValue: {
    fontWeight: 600,
  },
  root: {
    padding: 20,
    marginRight: 40,
    marginBottom: 30,
    borderRadius: 2,
    boxShadow: 'none',
  },
  question: {
    fontWeight: 400,
    marginTop: 10,
    fontSize: 14,
    color: colors.INTERFACE.ui7,
  },
  questionNumber: {
    width: 24,
    height: 24,
    marginTop: 3,
    borderRadius: 30,
    marginRight: 15,
    border: `1px solid ${colors.INTERFACE.ui3}`,
    textAlign: 'center',
    color: colors.INTERFACE.ui6,
  },
  questionsAnswered: {
    fontSize: 12,
    fontWeight: 500,
    marginTop: 25,
    color: '#ABABB6',
    textAlign: 'end',
  },
  questionTitle: {
    fontSize: 14,
    textTransform: 'uppercase',
    fontWeight: 'bold',
    letterSpacing: 1,
    color: colors.INTERFACE.ui6,
  },
  scrollList: {
    backgroundColor: '#F7F7F7',
  },
  textIcon: {
    color: colors.TEXT.normal,
    fontSize: 15,
    '&:hover': {
      color: colors.STATE.active,
    },
  },
  version: {
    fontWeight: 400,
    fontSize: 10,
    textTransform: 'uppercase',
    color: colors.INTERFACE.ui7,
  },
}));

export const photoScrollStyle = makeStyles(() => ({
  container: {
    maxHeight: 400,
    overflow: 'auto',
    paddingLeft: 40,
    paddingRight: 20,
  },
  text: {
    fontSize: 10,
    textDecoration: 'underline',
    color: colors.INTERFACE.ui7,
    '&:hover': {
      color: colors.STATE.active,
    },
  },
  icon: { fontSize: 13, marginBottom: -3, marginRight: 10 },
}));

export const singleChoiceBar = percentage =>
  makeStyles(() => ({
    barTextContainer: { marginTop: -38, paddingLeft: 15, paddingRight: 20 },
    barContainer: {
      backgroundColor: '#FFEBE3',
      display: 'inline-flex',
      height: 39,
    },
    container: {
      marginBottom: 6,
    },
    percentageContainer: {
      minWidth: 45,
    },
    percentageText: { color: '#4E4E56', fontWeight: 700, fontSize: 12 },
    percentageBar: {
      backgroundColor: '#FFC0A7',
      width: `${percentage}%`,
      height: 39,
    },
    responseContainer: { maxWidth: '85%' },
    responseText: { color: '#626262', fontWeight: 700, fontSize: 13 },
    numberOfResponses: { color: '#626262', fontSize: 13 },
  }));
