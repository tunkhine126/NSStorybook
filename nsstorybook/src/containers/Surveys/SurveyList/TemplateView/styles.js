import { makeStyles } from '@material-ui/core/styles';
import colors from 'global-styles';

export const styles = makeStyles(() => ({
  divider: {
    marginTop: 22,
    marginBottom: 47,
    marginRight: 40,
  },
  editBtn: { marginRight: 5 },
  editText: { marginRight: 5, fontSize: 13, fontWeight: 600 },
  moreBtn: {
    marginLeft: 15,
  },
}));

export const overviewStyles = makeStyles(() => ({
  table: {
    marginTop: 30,
    marginBottom: 30,
    boxShadow: 'none',
    '& .MuiTableCell-root': {
      fontSize: 14,
    },
    '& .MuiTableCell-head': {
      fontWeight: 900,
      color: colors.TEXT.normal,
    },
  },
  timeLaps: {
    fontSize: 12,
    color: colors.INTERFACE.ui5,
  },
  viewBtn: {
    textTransform: 'none',
  },
}));
