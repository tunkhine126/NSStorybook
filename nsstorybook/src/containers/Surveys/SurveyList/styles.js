import { makeStyles } from '@material-ui/core/styles';
import colors from 'global-styles';
import styled from 'styled-components';

export const surveyListStyles = makeStyles(() => ({
  actionBtns: {
    paddingRight: '16px',
    paddingBottom: '8px',
  },
  appBar: {
    backgroundColor: 'transparent',
    borderBottom: `1px solid ${colors.INTERFACE.ui4}`,
    boxShadow: 'none',
  },
  columnHeight: {
    height: '35px',
  },
  columnAction: {
    padding: 0,
    fontSize: '12px',
    textAlign: 'inherit',
  },
  divider: {
    marginLeft: '15px',
  },
  exportBtn: {
    width: '.7em',
    marginRight: '5px',
  },
  surveyListHeader: {
    marginBottom: '23px',
  },

  // Will clean up after this line when Survey preview is done
  activityContainer: {
    marginRight: '40px',
    flexBasis: '21%',
  },
  addMemberBtn: {
    padding: 0,
    margin: '15px 0 15px 15px',
  },
  caption: {
    fontSize: '10px',
    letterSpacing: '0.5px',
    color: colors.TEXT.light,
    fontWeight: 500,
  },
  container: {
    marginTop: '30px',
  },
  cardContainer: {
    maxWidth: '215px',
  },
  contentBody: {
    color: colors.INTERFACE.ui7,
  },
  captionSpacing: {
    marginBottom: '5px',
  },
  column1: {
    flexBasis: '31.05%',
  },
  column2: {
    flexBasis: '39.5%',
  },
  expand: {
    color: '#fff',
  },
  tab: {
    fontWeight: '700',
    color: `${colors.TEXT.medium}`,
    letterSpacing: '1px',
    marginRight: '20px',
  },
  surveyTemplate: {
    padding: 0,
    fontWeight: 600,
  },
  familyText: {
    textTransform: 'none',
    letterSpacing: '0',
    padding: 0,
    margin: 0,
    fontSize: '14px',
  },
  familiyName: {
    fontSize: '28px',
    letterSpacing: '1px',
    fontWeight: '600',
    fontFamily: 'Barlow, Roboto, Arial',
    color: colors.TEXT.normal,
    marginBottom: '8px',
  },
  familyId: {
    fontSize: '10px',
    letterSpacing: '.75px',
    textTransform: 'uppercase',
    color: colors.TEXT.light,
  },
  familyInfo: {
    marginTop: '15px',
  },
  fullWidth: {
    width: '100%',
  },
  familyCommunity: {
    marginTop: '-3px',
  },
  headerImg: {
    height: '153px',
    marginRight: '20px',
    borderRadius: 0,
    boxShadow: 'none',
  },
  recipientChildImg: {
    height: 91,
    '&.MuiCardActionArea-root': {
      width: 135,
    },
    '&:hover .img-action-text': {
      visibility: 'visible',
      opacity: 1,
    },
    '&.MuiCardActionArea-root:hover .MuiCardActionArea-focusHighlight': {
      opacity: 0.5,
    },
  },
  imgActionText: {
    position: 'absolute',
    top: 10,
    bottom: 0,
    left: 0,
    right: 0,
    color: '#fff',
    fontSize: 15,
    fontWeight: 600,
    textAlign: 'center',
    visibility: 'hidden',
    opacity: 0,
    zIndex: 1,
  },
  navContainer: {
    flex: 1,
    marginRight: '40px',
  },
  moreBtn: {
    marginLeft: '15px',
  },
  moveComplete: {
    backgroundColor: '#EAFFEC',
  },
  moveInProgress: {
    backgroundColor: '#fff7dd',
  },
  moveInContainer: {
    padding: '15px',
  },
  movedInDetails: {
    color: colors.TEXT.normal,
    marginTop: '15px',
    fontSize: '13px',
  },
  newSurveyBtn: {
    marginRight: '3px',
  },
  statusProgress: {
    backgroundColor: colors.INTERFACE.ui1,
    borderRadius: 2,
  },
  paper: {
    borderRadius: 2,
    boxShadow: 'none',
    marginBottom: '20px',
  },
  photoContainer: {
    fontSize: 12,
    color: colors.TEXT.light,
  },
  checkbox: {
    color: '#fff',
    margin: 'auto',
  },
  progressTip: {
    marginTop: '-1px',
  },
  twoColumns: {
    padding: '15px 0 15px 15px',
  },
  threeRowTitle: {
    fontSize: '14px',
    fontWeight: '600',
    color: colors.TEXT.normal,
  },
  survyReqBtn: {
    padding: '6px 18px 6px 5px',
  },
  selectedItemsContainer: {
    display: 'flex',
    alignItems: 'center',
    justifyContent: 'space-between',
    height: '40px',
    width: '100%',
    backgroundColor: colors.LINKS.link,
    boxShadow: '0px 2px 4px -3px rgba(88, 88, 88, 1)',
  },
  widgetHeader: {
    fontSize: '16px',
    fontWeight: 'bold',
    letterSpacing: '0.5px',
    marginBottom: '15px',
    color: colors.TEXT.normal,
  },
  widgetPadding: {
    padding: '15px 15px 0 15px',
  },
  upArrow: {
    transform: 'rotate(180deg)',
    width: 10,
  },
  upArrowBtn: {
    padding: 7,
    marginRight: 5,
    '&:hover.MuiIconButton-colorPrimary:hover svg path': {
      fill: colors.LINKS.link,
    },
  },
}));

export const DatePickerStyle = makeStyles(() => ({
  root: {
    maxWidth: 400,
    marginTop: 8,
    '& .MuiInputBase-root.Mui-focused': {
      border: `2px solid ${colors.LINKS.link}`,
    },
    '& .MuiFormLabel-root': {
      fontWeight: `normal`,
      color: colors.TEXT.light,
    },
    '& .MuiFormLabel-root.Mui-focused': {
      color: colors.LINKS.link,
    },
    '& .MuiFilledInput-root': {
      borderRadius: 2,
      backgroundColor: '#fff',
      border: `1px solid ${colors.INTERFACE.ui4}`,
    },
    '& .MuiFilledInput-underline:before, .MuiFilledInput-underline:after': {
      border: 'none',
    },
    '& .MuiFormHelperText-contained': {
      margin: '10px 0 8px 0',
    },
    '&:hover .MuiFilledInput-root': {
      borderColor: colors.LINKS.link,
      transition: 'border 0.3s',
    },
  },
}));

export const helperClasses = {
  survyReqBtn: {
    padding: '6px 18px 6px 5px',
  },
  lookUpBtn: {
    padding: '10px 0',
  },
  exportBtn: {
    padding: '6px 11px',
  },
  newSurveyBtn: {
    padding: '6px 15px 6px 5px',
  },
  selectorDimension: {
    width: 'auto',
    outterWidth: '107px',
    outterHeight: '0',
  },
  surveysCompleted: {
    outterWidth: '122px',
  },
  actionBtns: {
    padding: '7px 13px',
  },
  createBtn: {
    bg: colors.LINKS.link,
  },
  cancelBtn: {
    bg: 'transparent',
  },
};

export const progressBarStyles = color =>
  makeStyles(() => ({
    barColorPrimary: {
      backgroundColor: color,
    },
    root: {
      height: '20px',
    },
  }));

export const HeadOfHouseholdWrapper = styled.div`
  margin-bottom: 8px;
  .MuiFormLabel-root {
    font-weight: normal;
  }
`;

export const SearchIconStyle = styled.span`
  svg {
    fill: ${colors.TEXT.light};
  }
`;

export const UpArrowStyle = styled.span`
  svg path {
    fill: ${colors.TEXT.light};
  }
`;

export const SelectFormatStyle = styled.div`
  .MuiInputLabel-outlined {
    transform: translate(10px, 13px) scale(1);
  }
  .MuiInputLabel-outlined.MuiInputLabel-shrink {
    transform: translate(12px, 8px) scale(0.75);
  }
  .MuiInputLabel-outlined {
    color: ${colors.TEXT.light};
    font-weight: 400;
  }
  label {
    font-size: 12px;
    letter-spacing: 0;
  }
  input {
    font-size: 13px;
  }
`;

export const SurveyListStyle = styled.div`
  h2 {
    font-weight: 600;
    font-family: Barlow, Roboto, Helvetica, Arial, sans-serif;
  }
  margin-right: 40px;
`;

export const ExpansionPanelDetailsStyle = styled.div`
  .MuiExpansionPanelDetails-root {
    padding: 0 15px 24px;
  }
`;

export const ComponentStyle = styled.div`
  .MuiTab-root {
    min-width: 100px;
  }
`;

export const ThreeRowsStyle = styled.span`
  .MuiButton-root {
    text-transform: none;
    font-size: 14px;
  }
`;

export const TabContentStyle = styled.div`
  .MuiButton-root:hover {
    background-color: inherit;
  }
`;

export const ExpansionPanelStyle = styled.div`
  .MuiExpansionPanelSummary-expandIcon.Mui-expanded {
    transform: rotate(90deg);
  }
  .MuiExpansionPanelSummary-root {
    padding: 0 12px 0 15px;
    min-height: 73px;
  }
  .MuiExpansionPanel-root.Mui-expanded,
  .MuiExpansionPanelSummary-content.Mui-expanded {
    margin: 0;
  }
  .MuiPaper-elevation1 {
    box-shadow: none;
  }
  .MuiExpansionPanel-root:before {
    opacity: 0;
  }
  .MuiFormControl-marginNormal {
    margin-top: 0;
    height: 58px;
  }
`;
