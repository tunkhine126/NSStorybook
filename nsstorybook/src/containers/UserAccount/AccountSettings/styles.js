import { makeStyles } from '@material-ui/core/styles';

export const accSettingsStyles = makeStyles(() => ({
  langSetContainer: {
    minHeight: '188px',
    paddingTop: '25px',
    borderBottom: `1px solid #BBBBBB`,
  },
  passwordContainer: {
    minHeight: '80px',
    paddingTop: '25px',
  },
}));
