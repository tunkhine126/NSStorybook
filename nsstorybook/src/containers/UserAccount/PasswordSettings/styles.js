import { makeStyles } from '@material-ui/core/styles';
import styled from 'styled-components';

export const ConfirmPwdStyle = styled.div`
  .MuiFormControl-marginNormal {
    margin-top: 0;
  }
`;

export const pwdSettingsStyles = makeStyles(theme => ({
  confirmContainer: {
    paddingTop: '25px',
  },
  newPassword: {
    minHeight: '202px',
    borderBottom: '1px solid #BBBBBB',
    [theme.breakpoints.down('md')]: {
      minHeight: '225px',
    },
  },
}));
