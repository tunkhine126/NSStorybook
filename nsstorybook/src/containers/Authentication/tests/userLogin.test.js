/**
 *
 * Tests for Authentication
 *
 * @see https://github.com/react-boilerplate/react-boilerplate/tree/master/docs/testing
 *
 */

import React from 'react';
import PropTypes from 'prop-types';
import { Provider } from 'react-redux';
import { browserHistory } from 'react-router-dom';
import { render } from '@testing-library/react';
import { IntlProvider } from 'react-intl';
import { withStyles, makeStyles } from '@material-ui/core/styles';
// import 'jest-dom/extend-expect'; // add some helpful assertions
import { ApolloProvider } from 'react-apollo';

import client from 'utils/apollo-client';

import { DEFAULT_LOCALE } from 'i18n';
import configureStore from 'configureStore';
import UserLogin from '../userLogin';

const style = makeStyles(theme => ({
  form: {
    [theme.breakpoints.down('sm')]: {
      marginLeft: '40px',
      marginRight: '40px',
    },
  },
}));

const history = {
  location: { pathname: '/dashboard' },
};

const Composer = ({ classes }) => (
  <UserLogin
    classes={classes}
    history={history}
    signInUser={() => {}}
    setmessageState={() => ({
      message: 'Logged Out',
      open: true,
      type: 'success',
    })}
  />
);

const UserLoginComposition = withStyles(style)(Composer);

Composer.propTypes = {
  classes: PropTypes.object.isRequired,
};

describe('<UserLogin />', () => {
  let store;

  beforeAll(() => {
    store = configureStore({}, browserHistory);
  });

  it('Expect to not log errors in console', () => {
    const spy = jest.spyOn(global.console, 'error');
    render(
      <Provider store={store}>
        <IntlProvider locale={DEFAULT_LOCALE}>
          <ApolloProvider client={client}>
            <UserLoginComposition />
          </ApolloProvider>
        </IntlProvider>
      </Provider>
    );
    expect(spy).not.toHaveBeenCalled();
  });

  it('Should render and match the snapshot', () => {
    const {
      container: { firstChild },
    } = render(
      <Provider store={store}>
        <IntlProvider locale={DEFAULT_LOCALE}>
          <ApolloProvider client={client}>
            <UserLoginComposition />
          </ApolloProvider>
        </IntlProvider>
      </Provider>
    );
    expect(firstChild).toMatchSnapshot();
  });
});
