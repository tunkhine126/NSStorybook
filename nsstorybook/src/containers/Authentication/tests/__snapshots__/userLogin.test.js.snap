// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<UserLogin /> Should render and match the snapshot 1`] = `
<form>
  <div
    class="MuiGrid-root Composer-form-276 MuiGrid-item MuiGrid-grid-xs-12"
  >
    <div
      class="MuiFormControl-root MuiTextField-root MuiFormControl-marginNormal MuiFormControl-fullWidth"
    >
      <label
        class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated"
        data-shrink="false"
        for="email"
        id="email-label"
      >
        <span>
          Email*
        </span>
      </label>
      <div
        class="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-fullWidth MuiInput-fullWidth MuiInputBase-formControl MuiInput-formControl"
      >
        <input
          aria-invalid="false"
          class="MuiInputBase-input MuiInput-input"
          id="email"
          name="email"
          type="email"
          value=""
        />
      </div>
    </div>
    <div
      class="MuiFormControl-root MuiTextField-root MuiFormControl-marginNormal MuiFormControl-fullWidth"
    >
      <label
        class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated"
        data-shrink="false"
        for="password"
        id="password-label"
      >
        <span>
          Password*
        </span>
      </label>
      <div
        class="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-fullWidth MuiInput-fullWidth MuiInputBase-formControl MuiInput-formControl"
      >
        <input
          aria-invalid="false"
          class="MuiInputBase-input MuiInput-input"
          id="password"
          name="password"
          type="password"
          value=""
        />
      </div>
    </div>
  </div>
  <div
    class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-direction-xs-column MuiGrid-align-items-xs-center MuiGrid-justify-xs-center MuiGrid-grid-xs-12"
  >
    <label
      class="MuiFormControlLabel-root"
    >
      <span
        aria-disabled="false"
        class="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-449 MuiCheckbox-root MuiCheckbox-colorPrimary MuiIconButton-colorPrimary"
      >
        <span
          class="MuiIconButton-label"
        >
          <input
            class="PrivateSwitchBase-input-452"
            data-indeterminate="false"
            type="checkbox"
            value=""
          />
          <svg
            aria-hidden="true"
            class="MuiSvgIcon-root"
            focusable="false"
            role="presentation"
            viewBox="0 0 24 24"
          >
            <path
              d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
            />
          </svg>
        </span>
        <span
          class="MuiTouchRipple-root"
        />
      </span>
      <span
        class="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
      >
        <span>
          Remember Me
        </span>
      </span>
    </label>
    <button
      class="MuiButtonBase-root MuiButton-root MuiButton-contained undefined undefined undefined MuiButton-containedPrimary MuiButton-containedSizeLarge MuiButton-sizeLarge"
      tabindex="0"
      type="submit"
    >
      <span
        class="MuiButton-label"
      >
        <span>
          Log In
        </span>
      </span>
      <span
        class="MuiTouchRipple-root"
      />
    </button>
    <a
      class="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorPrimary"
      href="/"
    >
      <span>
        Forgot your password?
      </span>
    </a>
    <a
      class="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorPrimary"
      href="/"
    >
      <span>
        Didn't receive unlock instructions?
      </span>
    </a>
  </div>
</form>
`;
